### ISSUE TITLE
Insufficient color contrast ratio for error messages

### ISSUE THREAD

**Issue by meganvilla** [OpenSpace/OpenSpace#3103]
Error messages, e.g. "Profile name must be specified", have red text (#DD1111) which has an insufficient color contrast ratio of 4.4:1 with the gray background (#F0F0F0).

**Suggested Solution**: Use a darker red color, e.g. #D90D0D which has 4.6:1 color contrast with the gray background.

Severity level 2 from PAC
